packages/icon/src/vaadin-icon.js

```diff
@@ -325,8 +345,9 @@ class Icon extends ThemableMixin(ElementMixin(ControllerMixin(SlotStylesMixin(Ic
     // https://github.com/vaadin/web-components/issues/6301
     this.icon = '';
 
-    if (src.includes('#')) {
-      this.svg = svg`<use href="${src}"/>`;
```

Comment:
After removing this, the import to `import { svg } from 'lit';` became unused

Reply:
Removed.